lib/SIL/IR/SILDebugInfoExpression.cpp

```diff
@@ -0,0 +1,27 @@
+//===--- SILDebugInfoExpression.cpp - DIExpression for SIL ------*- C++ -*-===//
```

Comment:
The `-*- C++ -*-` marker only makes sense in a `.h` file, where it's ambiguous whether the language is C or C++ based on the file name.